### ISSUE TITLE
[WIP] Inpaint in GUI

### ISSUE THREAD

**Issue by fszontagh** [fszontagh/sd.cpp.gui.wx#23]
Implement inpaint if the [PR accepted](https://github.com/leejet/stable-diffusion.cpp/pull/511) in sd.cpp

**Comment by fszontagh** [comment]
![stable-diffusion-gui_inpaint_720_fps30](https://github.com/user-attachments/assets/a0c02eff-11da-487e-a09b-9448d18006f8)

**Comment by Amin456789** [comment]
any release date? im so excited for inpaint and outpaint
a suggeston= could u please release zip version of instead exe too for installing again like before, for more portable. also it will be great if u can make all the settings foldes go to the same folder as installtion folder of sd itself for becoming more portable . cause i saw settings go to ducuments folder.thnx

**Comment by fszontagh** [comment]
> any release date? im so excited for inpaint and outpaint

Yeah, i'm already working on the next release, but the 2d painting have some issues, i had to change the entire GUI layout to fit in the window on more desktop resolutions. And i had issues with the 2d painting  itself on Windows OS. (What is worked on linux that's not worked on windows, of cousre :) ) 

> could u please release zip version of instead exe too for installing again like before, for more portable

On the next release i will not, thats need a lot of testing. The current github release workflow is works and i don't want to modify it yet. But after the next release i may will give a try. 

> also it will be great if u can make all the settings foldes go to the same folder as installtion folder of sd itself for becoming more portable . cause i saw settings go to ducuments folder.thnx


On windows, the settings are stored in registry, while on linux is stored in config files. It's not safe to transfer the settings between PCs.

**Comment by Amin456789** [comment]
great tnq

**Comment by fszontagh** [comment]
@Amin456789 i just [release the new version](https://github.com/fszontagh/sd.cpp.gui.wx/releases/tag/v0.2.4)
Added the inpaint / outpaint feature, similar to the animated gif, but may better :) 
Please check it out, i close this issue.
